### ISSUE TITLE
[ENH] Collection normaliser transformer

### ISSUE THREAD

**Issue by TonyBagnall** [aeon-toolkit/aeon#2000]
### Describe the feature or idea you want to propose

when writing notebooks, it would have been helpful to have a normaliser BaseCollectionTransformer that could z-normalise, standardise or min/max scale, even as a wrapper of numpy, since it needs to be done on the correct axis etc.

### Describe your proposed solution

something like 

```python
from aeon.transformations.collection import Normalise
norm = Normalise()
xt = norm.fit_transform(x)
norm = Normalise(method="min_max")
xt = norm.fit_transform(x)
```

**Comment by aryanpola** [comment]
So a new class Normalise is needed in aeon.transformations.collection?

**Comment by aryanpola** [comment]
@aeon-actions-bot assign @aryanpola

**Comment by MatthewMiddlehurst** [comment]
Is this different from `TimeSeriesScaler` @TonyBagnall?

**Comment by TonyBagnall** [comment]
done in 2005
